tests/unit/test-class-wc-payments-account.php

```diff
@@ -3174,6 +3174,73 @@ public function test_get_tracking_info() {
 		$this->assertSame( $expected, $this->wcpay_account->get_tracking_info() );
 	}
 
+	public function test_get_recommended_payment_methods_unsupported_country() {
+		$this->assertSame( [], $this->wcpay_account->get_recommended_payment_methods( 'unsupported_country' ) );
+	}
+
+
```

Comment:
No need for two empty lines.

Reply:
changed in [3af3993](https://github.com/Automattic/woocommerce-payments/pull/9825/commits/3af399332f1bc9159887a0f09db97ea47fa75fcb). thx